data/etc/avahi/services/freedombox.service

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" standalone='no'?>
+<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
+<service-group>
+  <name replace-wildcards="yes">%h</name>
+
+  <service>
+    <type>_freedombox._tcp</type>
+    <port>80</port>
+    <host>%h</host>
+  </service>
```

Comment:
We could add (and manage) information related to the following:
- Primary domain that this machine should referred to as.
- Whether FreedomBox has been configured (first-boot completed).
- Whether FreedomBox has a proper SSL certificate configured.